pyproject.toml

```diff
@@ -83,7 +83,7 @@ dependencies = [
     # The usage was added in #30596, seemingly only to override and improve the default error message.
     # Either revert that change or find another way, preferably without using connexion internals.
     # This limit can be removed after https://github.com/apache/airflow/issues/35234 is fixed
-    "connexion[flask]>=2.10.0,<3.0",
+    "connexion[flask,uvicorn,swagger-ui]>=3.0",
```

Comment:
Right now we [install swagger via npm](https://github.com/apache/airflow/pull/28788) package [`swagger-ui-dist`](https://www.npmjs.com/package/swagger-ui-dist) and attach it to the Connexion, so we should do one of the action:
- Remove `swagger-ui-dist` and and use [`swagger-ui-bundle`](https://github.com/spec-first/swagger_ui_bundle) instead. In this case dependecy bot do not show should we update swagger version or not, see: https://github.com/apache/airflow/pull/28824
- Do not use [swagger-ui-bundle](https://github.com/spec-first/swagger_ui_bundle) in this case we need to provide path to the swagger ui into the Connexion options

Reply:
Thanks for the feedback. 

I believe this is related to "Properly setup bundled swagger-ui in Connexion #36975". 
When I opened the swagger page on this branch, I got the response `{"type": "about:blank", "title": "Not Found", "detail": "The requested URL was not found on the server. If you entered the URL manually please check your spelling and try again.", "status": 404}`

We will make task 4 about this and will work on it later.

Reply:
It doesn't work with current implementation with `swagger-ui-dist` because is should be use `swagger_ui_template_dir` not a `swagger_ui_path`: https://connexion.readthedocs.io/en/stable/swagger_ui.html#connexion.options.SwaggerUIOptions

Reply:
Hey @Satoshi-Sh -> I looked at the history of swaggers, and I think we should attempt to update the swagger-ui-dist instead (sorry :(  - I hope setting up the things in https://github.com/sudiptob2/airflow/pull/8 and the hint from @Taragolis abobve - will help configuring the dist one. 

The thing is that the one that we can install from PyPI is already pretty old - and it's only a question of time when some vulnerabilities will be found in it. If you look here - https://pypi.org/project/swagger-ui-bundle/1.1.0/#files the one from November bundles the javascript for swagger `4.15.5` which has been released a year ago https://www.npmjs.com/package/swagger-ui-dist/v/4.15.5 where we are currently at https://www.npmjs.com/package/swagger-ui-dist/v/5.11.8 released 7 days ago. So if we could attempt to use the sdist one - that is way better. However it might turn  out that there will be a compatibility problem with the connexion and 5.* line of swagger - @RobbeSneyders - maybe you can comment on that - do you foresee any problem with attempting to use swager-ui-dist  5.* version with Connexion 3 ?

Reply:
Thanks for the pointer Jarek. I will have a look at it.

Reply:
I made another [PR](https://github.com/sudiptob2/airflow/pull/13)

It's working fine other than the Redoc page. It cannot load favicon.ico. I need to investigate about it. 

I upgraded the swagger-ui-dist to 5.11.8. It seems working fine as before.

Reply:
Sounds great. I think you can even merge that one into @sudiptob2 PR and we can solve the `favicon.ico` later.